### PR TITLE
[JBPM-9557] Enable sb keycloak test with testcontainers

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**: [JBPM-9557](https://issues.redhat.com/browse/JBPM-9557)

referenced Pull Requests:

- https://github.com/kiegroup/droolsjbpm-integration/pull/2368

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4059,6 +4059,11 @@
         <version>${version.org.jsoup}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.keycloak</groupId>
+        <artifactId>keycloak-admin-client</artifactId>
+        <version>${version.org.keycloak}</version>
+      </dependency>
```
Comment (mareknovotny) [inline pom.xml:4066]: this is not in test scope intentionally?

Comment (gmunozfe) [reply pom.xml:4067]: added test scope

**Review by afalhambra — APPROVED** [review]
Looks good to me. Good job

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
@gmunozfe can you please follow the template PR nextime? That is for related/dependent PR links in first comment ;) thanks in advance

**Comment by mareknovotny** [comment]
jenkins do fdb

**Review by mareknovotny — APPROVED** [review]

**Comment by mareknovotny** [comment]
2 failed tests are from DMN editors not related to this change set

**Pull request merged**